terminal.go

```diff
@@ -22,5 +23,10 @@ func colorEnabledWriter(w io.Writer) (io.Writer, bool) {
 	if !isatty.IsTerminal(f.Fd()) {
 		return w, false
 	}
+	// Check the TERM environment variable specifically
+	// to check for "dumb" terminals.
+	if os.Getenv("TERM") == "dumb" {
```

Comment:
Given that you're now checking an ENV variable, don't you need to isolate the test suite so someone running it in a "TERM=dumb" terminal doesn't get spurious failures?